pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -214,6 +216,19 @@ stdenv.mkDerivation {
 
   enableParallelBuilding = true;
 
+  # Install linker plugin to make 'ar', 'ld' and friends auto-load
+  # linker plugin to handle LTO bytecode without explicit --plugin
+  # parameter.
+  #
+  # We can install the link only if our compiler's host and target
+  # match binutils' host and target.
+  postInstall = lib.optionalString (   stdenv.cc.stdenv.hostPlatform == stdenv.hostPlatform
+                                    && stdenv.cc.stdenv.targetPlatform == stdenv.targetPlatform) ''
```

Comment:
```suggestion
  postInstall = lib.optionalString (stdenv.cc.stdenv.hostPlatform == stdenv.hostPlatform
                                    && stdenv.cc.stdenv.targetPlatform == stdenv.targetPlatform) ''
```

Reply:
Please stop with these formatting suggestions, it's personal preference, you do no maintenance work on binutils. 

Until nixpkgs gets a formatter enforced treewide, it's just review noise that you're creating.

Please, stop.